libs/community/langchain_google_community/drive.py

```diff
@@ -276,6 +278,11 @@ def _load_credentials(self) -> Any:
         if self.token_path.exists():
             creds = Credentials.from_authorized_user_file(str(self.token_path), SCOPES)
 
+        if self.credentials:
```

Comment:
nits: maybe we could move it a few lines up to avoid a not needed `Credentials.from_authorized_user_file` execution?

Reply:
My thought was the other options wouldn't be called since they don't/wouldn't exist if you just pass in credentials? i.e. I'd set it on init to only use credentials and no other option: 
```
drive_loader = GoogleDriveLoader(credentials=credentials, folder_id='someid', recursive=True)
```

Reply:
OK, looks like trailing spaces. Fixed.